liteeth/mac/__init__.py

```diff
@@ -37,7 +38,13 @@ def __init__(self, phy, dw,
             self.rx_slots  = CSRConstant(nrxslots)
             self.tx_slots  = CSRConstant(ntxslots)
             self.slot_size = CSRConstant(2**bits_for(eth_mtu))
-            self.submodules.interface = LiteEthMACWishboneInterface(32, nrxslots, ntxslots, endianness)
+            self.submodules.interface = LiteEthMACWishboneInterface(
+                32,
+                nrxslots,
+                ntxslots,
+                endianness,
+                timestamp_source
+            )
```

Comment:
If we switch to multi-lines, can we specify the parameters: `dw=32`, `nrxslots= nrxslots`?